docs/section18/Rule18-8.md

```diff
@@ -0,0 +1,41 @@
+
+# CHW&CW - Rule 18-8  
+
+**Rule ID:** 18-8  
+**Rule Description:** For systems using purchased chilled water, the cooling source shall be modeled as purchased chilled water in both the proposed design and baseline building design. If any system in the proposed design uses purchased chilled water, all baseline systems with chilled water coils shall use purchased chilled water. On-site chillers and direct expansion equipment shall not be modeled in the baseline building design.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.1.1 & G3.1.1.3.1 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. P-RMR is modeled with purchased chilled water.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:**  
+
+1. check_purchased_chw_hhw()
+2. get_baseline_system_types()
+
+**Applicability Checks:**
+
+- Check if P-RMR is modeled with purchased chilled water or purchased hot water/steam: `purchased_chw_hhw_status_dict = check_purchased_chw_hhw(P_RMR)`
+
+  - If P-RMR is not modeled with purchased chilled water, rule is not applicable to B-RMR: `if NOT purchased_chw_hhw_status_dict["PURCHASED_COOLING"]: RULE_NOT_APPLICABLE`
+
+  - Else, P-RMR is modeled with purchased chilled water, continue to rule logic: `else: CHECK_RULE_LOGIC`
+
+## Rule Logic:  
+
+- Check if B-RMR is modeled with any air-side system that is Type-1, 2, 3, 4, 5, 6, 7, 8, 9, 11, 12, 13, 1b, 3b, 5b, 6b, 7b, 8b, 11b, 12b, 13b, i.e. any air-side system using cooling source other than purchased chilled water, set check flag to True: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-1", "SYS-2", "SYS-3", "SYS-4", "SYS-5", "SYS-6", "SYS-7", "SYS-8", "SYS-9", "SYS-11", "SYS-12", "SYS-13", "SYS-1B", "SYS-3B", "SYS-5B", "SYS-6B", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): check_flag = TRUE`
+
+**Rule Assertion:**
+
+- Case 1: If check flag is True: `if check_flag: FAIL`
+
```

Comment:
@xingcx I had the same question for 21-1 - not sure what this statement means If check flag is True: `if check_flag: FAIL`

Reply:
@xingcx the same comment above is still confusing. Now I understand that check_flag = true means that the baseline has been modeled with a system that is now PCHW- hence it fails. Could we change 'Check_flag' to something more intuitive like 'non-pchw-system'
then it would be if non_P-CHW_system = TRUE, then FAIL
if non_P-CHW_systeem = FALSE, then PASS

Reply:
Updates required, the check is still confusing. I will approve, the RCT team should correct this during implementation.